### PR TITLE
Added better error handling for command line arguments.

### PR DESCRIPTION
**Pull request by Phlosioneer**
After debugging a buildscript I added to my IDE for several hours, I figured out what was wrong with it. There were issues with the parameter passing. I was using this project to test the script, so I ended up writing error-handling code. So I cleaned up a bit and make this PR, to save you a bit of time. You can decline if you want.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -22,24 +23,66 @@ use std::env;
...
+            println!("Unable to open file \"{}\" from current directory \"{}\": {}",
+		             path.as_ref().display(), 
+		             Path::new(".").canonicalize().unwrap().display(),
+		             e);
+            return None;
+        }
+    };
+	
+	// Create a box without flooding the stack.
+	let pif_size = mem_map::PIF_ROM_LENGTH as usize;
```
Comment (pepyakin) [inline src/main.rs]: Looks like there are tabs crept in

Comment (Phlosioneer) [inline src/main.rs]: Oops! I'll fix it in a bit.

Comment (Phlosioneer) [inline src/main.rs]: Fixed: 6196112

**Comment by yupferris** [comment]
This is rad, but yeah, it kinda falls under the category of stuff I'd rather get live on the stream. Also, half of this would be invalid anyways due to https://github.com/yupferris/rustendo64/issues/25. But hey, glad you were able to debug more of your editor with it, that's pretty sick :)

**Pull request closed without merging**